webapp-django/crashstats/crashstats/context_processors.py

```diff
@@ -0,0 +1,7 @@
+from django.conf import settings
+
+
+def help_urls(request):
+    return {
+        'GOOGLE_AUTH_HELP_URL': settings.GOOGLE_AUTH_HELP_URL,
+    }
```

Comment:
With the socorro webapp, do we add `settings` to the context for templates? If so, then we don't need this context processor.